packages/app/src/components/inspector/SymbolSourceRow.tsx

```diff
@@ -0,0 +1,92 @@
+import Sketch from '@sketch-hq/sketch-file-format-ts';
+import { Spacer } from 'noya-designsystem';
+import { memo } from 'react';
+import ColorInputFieldWithPicker from './ColorInputFieldWithPicker';
+import * as InspectorPrimitives from './InspectorPrimitives';
+
+interface Props {
+  resizesContent: boolean;
+  backgroundColor: Sketch.Color;
+  hasBackgroundColor: boolean;
+  includeBackgroundColorInExport: boolean;
+  includeBackgroundColorInInstance: boolean;
+  setAdjustContentOnResize: (value: boolean) => void;
+  setHasBackgroundColor: (value: boolean) => void;
+  setBackgroundColor: (value: Sketch.Color) => void;
+  setIncludeBgInExport: (value: boolean) => void;
```

Comment:
We don't abbreviate "background" as "bg" anywhere else, so for consistency, we shouldn't abbreviate it here or in our actions